src/http/http_1.rs

```diff
@@ -8,39 +8,31 @@ use crate::cli::CLIError;
 
 pub async fn start_http_1(sc: Arc<ServerConfig>) -> std::prelude::v1::Result<(), anyhow::Error> {
   let addr = sc.addr();
-  Ok(
-    sc.tokio_runtime()?
-      .spawn(async move {
-        let sc_cloned = sc.clone();
+  let sc_cloned = sc.clone();
```

Comment:
since we're not moving `sc`, we don't need to clone it; we can use the original value

```suggestion
  let sc_cloned = sc.clone();
```